.github/workflows/update-flakes.yml

```diff
@@ -0,0 +1,49 @@
+name: go-licenses
```

Comment:
go-update-flakes?

Reply:
Bah, I'm bad at this copying and pasting. Fixed.